support-frontend/assets/components/orderSummary/orderSummary.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { SvgInfo } from '@guardian/src-icons';
+import typeof GridImageType from 'components/gridImage/gridImage';
+import { type GridImg } from 'components/gridImage/gridImage';
+import * as styles from './orderSummaryStyles';
+
+type MobileOrderSummary = {
+  title: string,
+  price: string,
+}
+
+type PropTypes = {
+  children: Node,
+  changeSubscription?: string | null,
+  image: $Call<GridImageType, GridImg>,
+  mobileSummary: MobileOrderSummary,
+  total: string,
+};
+
+function OrderSummary(props: PropTypes) {
+  return (
+    <aside css={styles.wrapper}>
+      <div css={styles.topLine}>
+        <div css={styles.topLineBorder}>
+          <h3 css={styles.title}>Order summary</h3>
+          <a href={props.changeSubscription}>Change</a>
```

Comment:
Do we still want to render this link in the event `props.changeSubscription` is an empty string?

Reply:
Good point, we probably don't.